src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -30,6 +30,8 @@ class Definition
     private $factoryMethod;
     private $factoryService;
     private $shared = true;
+    private $deprecated = false;
+    private $deprecationTemplate = 'The "%service_id%" service is deprecated. You should stop using it, as it will soon be removed.';
```

Comment:
we shoul either name this deprecationMessage, or rename getDeprecationMessage to getDeprecationTemplate. MIxing both vocabularies can be confusing